reporter/src/test/kotlin/reporters/OpossumReporterTest.kt

```diff
@@ -176,8 +176,8 @@ class OpossumReporterTest : WordSpec({
         }
 
         "create signals with all uuids being assigned" {
-            opossumInput.pathToSignal.forEach { e ->
-                e.value.forEach { uuid ->
+            opossumInput.pathToSignal.forEach { (_, signal) ->
```

Comment:
Would it make sense to only iterate over the `values` here, similar to like you did in a previous change for `keys`?

Reply:
And actually you do a similar change in the test in the next commit.

Reply:
Yeah, the change became necessary because `forAll` isn'T available for `Map`s.

Reply:
I don't see this being addressed. IMO it would still make sense to align on

    opossumInput.pathToSignal.values.forEach { signal ->

here.

Reply:
Of course it would make sense. But is it worth the effort?
I doubt it but will do anyway.